core/shortcut_items.ts

```diff
@@ -188,8 +192,27 @@ export function registerPaste() {
       return !workspace.options.readOnly && !Gesture.inProgress();
     },
     callback() {
-      if (!copyData || !copyWorkspace) return false;
-      return !!clipboard.paste(copyData, copyWorkspace);
+      if (!copyData || !copyCoords || !copyWorkspace) return false;
+      const {
```

Comment:
For clarity, please update these names to `viewportLeft`, `viewportTop`, etc. That helps the next person who reads this!